### PR TITLE
[REF] autopep8: Delete vim comment

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

File: tools/autopep8_extended.py [hunk 1]
```diff
@@ -20,8 +20,29 @@ def __init__(self, pep8_options, source):
...
+        'Detect vim comment'
+        msg_code = 'CW0002'
+        check_result = []
+        msg = self.msgs[msg_code]
+        line_no = 0
+        for line in self.source:
+            column = 0
+            line_no += 1
+            line = line.strip()
+            if line.startswith('#') and \
```
Comment (pedrobaeza) [inline tools/autopep8_extended.py]: Why don't use here a regular expression?

Comment (moylop260) [inline tools/autopep8_extended.py]: In this case regex is a good idea... line.startwith too.
I saw a post of problems with regex recently...
I prefered avoid them in this simple case.

Comment (pedrobaeza) [inline tools/autopep8_extended.py]: OK then

**Comment by pedrobaeza** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Comment by max3903** [comment]
@moylop260 can you fix the conflict so we can merge it ? Thank you!

**Comment by moylop260** [comment]
@max3903 
rebased

**Pull request merged**